defender/staged/downloaded/Forta_Low_Liquidity.js

```diff
@@ -134,16 +134,23 @@ exports.handler = async function (autotaskEvent) {
   const {
     compTokenSymbol,
     maliciousAddress,
+    protocolVersion,
   } = metadata;
   if (maliciousAddress === undefined) {
     throw new Error('maliciousAddress undefined');
   }
 
+  // Handle older alerts which don't specify the protocol version
+  let versionString = '';
+  if (protocolVersion !== undefined) {
+    versionString = ` (Compound v${protocolVersion})`;
+  }
+
```

Comment:
This can be hardcoded to ` (Compound v2)`. The Low Liquidity Bot checks for a condition that can only exist on Compound V2 contracts.

Reply:
I'm not sure about this.   I do realize that this and many of the other monitors only relate to Compound v2, but the `protocolVersion` is coming from the Forta bot itself which has full knowledge of the relationship between alert and protocol version.  In these cases the `protocolVersion` is hardcoded in the Forta bots.

Reply:
I see the argument. This Autotask is really just a pass through for the Bot, which is our "source of truth." This logic acknowledges that fact, rather than assigning its own alert characteristics (which would seemingly come from nowhere).